.github/workflows/data_quality_outputs.yml

```diff
@@ -0,0 +1,34 @@
+#
+#  1. Check we get expected outputs from data quality parameters on sample data
+#
+name: data_quality_outputs
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  parsing:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2 
+
+      - name: Compile Parser
+        run: javac -cp java/JTransforms-3.1-with-dependencies.jar java/*.java
+        
+      
+      - name: Load data
+        run: bash utilities/downloadDataModels.sh
+       
+      - name: Summarise sample
+        run: |
+            import accelerometer
+            python accelerometer/summariseEpoch.py data/sample.cwa
+          
+      - name: compare output
```

Comment:
Actually, for the comparison, you can just save a sample summarize epcoh file and then using the below snippet to compare that with the generated one 

```python
      - name: compare epoch output
         uses: tianhaoz95/mirror-action@v1.0.1
         with:
           original: PATH2ORIGINAL
           mirror: PATH2GENERATED
```